src/wpool_queue_manager.erl

```diff
@@ -48,6 +48,7 @@
 -record(state, { wpool                 :: wpool:name()
                , clients               :: queue:queue({cast|{pid(), _}, term()})
                , workers               :: gb_sets:set(atom())
+               , monitors              :: gb_trees:tree(atom(), {reference(), {pid(), any()}})
```

Comment:
Line 51 is too long:                , monitors              :: gb_trees:tree(atom(), {reference(), {pid(), any()}}).